tools/logger/logger.c

```diff
@@ -444,6 +444,10 @@ int main(int argc, char *argv[])
 	if (config.input_std) {
 		config.in_fd = stdin;
 	} else if (baud) {
+		if (!config.in_file) {
+			ret = -EINVAL;
+			goto out;
+		}
```

Comment:
I've seen another change to this part of the file already (similar in spirit, even). Try not to duplicate work!